example/README.md

```diff
@@ -227,38 +229,40 @@ In the event that you are running the tests outside of the Cumulus sandbox envir
 
 ### Adding tests for an existing workflow
 
-Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and supporting JSON files can go in there.
+Workflow tests are located in the `/spec/<workflow-name>` directory. Any tests and supporting JSON files can go in there.
```

Comment:
Sorry to keep nitpicking, especially since these bad docs aren't your fault, but this isn't true. The workflow tests are either in can be in spec/serial or spec/parallel depending on whether they can be run in parallel or not

Reply:
No worries, I will update that!